.github/workflows/deploy.yml

```diff
@@ -42,14 +42,18 @@ jobs:
         run: |
           invoke virtualenv
           invoke run -t update-downloads
-          invoke run -t update-docs
+
+      # - name: Site initialization
+      #   run: |
+      #     invoke virtualenv
+      #     invoke run -t update-downloads
+      #     invoke run -t update-docs
 
       - name: 🔧 Build HTML
         run: |
           invoke doc.clean
           invoke doc
           invoke intl -l zh_CN
```

Comment:
The post processing is where a lot of fixups are to be done for the website, please do not remove it, was it causing problems?

Reply:
layout of the web page is broken.

Reply:
Yes, I will address that though. Are you saying it's not broken before the post processing? With proper base URL specified, that may well be the case. What way would `post-process` learn the base URL, by looking at the canonical URL in the content I guess? Do you specify the Github base URL yet?

Reply:
The conf. Py specifies

```python
html_baseurl = blog_baseurl
` ` `